### PR TITLE
feat(distributed): Add Readme

### PR DESCRIPTION
**Pull request by zhenlei520**

### CODE REVIEW

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.Azure/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## Masa.Contrib.Data.DistributedLocking.Medallion.Azure
+
+### Example:
+
+```c#
+Install-Package Masa.Contrib.Data.DistributedLocking.Medallion.Azure
+```
+
+1. 修改类`Program`
```
Comment (doddgu) [inline src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.Azure/README.md:11]: English?

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.FileSystem/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Masa.Contrib.Data.DistributedLocking.Medallion.FileSystem
+
+### Example:
+
+```c#
+Install-Package Masa.Contrib.Data.DistributedLocking.Medallion.FileSystem
+```
+
+1. 修改类`Program`
```
Comment (doddgu) [inline src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.FileSystem/README.md:11]: English?

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.MySql/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## Masa.Contrib.Data.DistributedLocking.Medallion.MySql
+
+### Example:
+
+```c#
+Install-Package Masa.Contrib.Data.DistributedLocking.Medallion.MySql
+```
+
+1. 修改类`Program`
```
Comment (doddgu) [inline src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.MySql/README.md:11]: English?

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.Oracle/README.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+## Masa.Contrib.Data.DistributedLocking.Medallion.Oracle
+
+### Example:
+
+```c#
+Install-Package Masa.Contrib.Data.DistributedLocking.Medallion.Oracle
+```
+
+1. 修改类`Program`
```
Comment (doddgu) [inline src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.Oracle/README.md:11]: English?

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.PostgreSql/README.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+## Masa.Contrib.Data.DistributedLocking.Medallion.PostgreSql
+
+### Example:
+
+```c#
+Install-Package Masa.Contrib.Data.DistributedLocking.Medallion.PostgreSql
+```
+
+1. 修改类`Program`
```
Comment (doddgu) [inline src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.PostgreSql/README.md:11]: English?

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.Redis/README.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+## Masa.Contrib.Data.DistributedLocking.Medallion.Redis
+
+### Example:
+
+```c#
+Install-Package Masa.Contrib.Data.DistributedLocking.Medallion.Redis
+```
+
+1. 修改类`Program`
```
Comment (doddgu) [inline src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.Redis/README.md:11]: English?

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.SqlServer/README.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+## Masa.Contrib.Data.DistributedLocking.Medallion.SqlServer
+
+### Example:
+
+```c#
+Install-Package Masa.Contrib.Data.DistributedLocking.Medallion.SqlServer
+```
+
+1. 修改类`Program`
```
Comment (doddgu) [inline src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.SqlServer/README.md:11]: English?

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.WaitHandles/README.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+
+## Masa.Contrib.Data.DistributedLocking.Medallion.WaitHandles
+
+### Example:
+
+```c#
+Install-Package Masa.Contrib.Data.DistributedLocking.Medallion.WaitHandles
+```
+
+1. 修改类`Program`
```
Comment (doddgu) [inline src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.WaitHandles/README.md:11]: English?

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.ZooKeeper/README.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+
+## Masa.Contrib.Data.DistributedLocking.Medallion.SqlServer
+
+### Example:
+
+```c#
+Install-Package Masa.Contrib.Data.DistributedLocking.Medallion.SqlServer
+```
+
+1. 修改类`Program`
```
Comment (doddgu) [inline src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.ZooKeeper/README.md:11]: English?

**Review by doddgu — CHANGES_REQUESTED** [review]

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Local/README.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+1. Modify the class `Program`
+
+``` C#
+builder.Services.AddLocalDistributedLock();
+```
+
+2. Use distributed locks
+
+``` C#
+IDistributedLock distributedLock;//从DI获取`IDistributedLock`
```
[the remark this answers is not in the document]
Comment (zhenlei520) [reply src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Local/README.md]: processed

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Local/README.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+using (var lockObj = distributedLock.TryGet("Replace You Lock Name"))
```
[the remark this answers is not in the document]
Comment (zhenlei520) [reply src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Local/README.md]: processed

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.Azure/README.md (hunk 1 above)
Comment (zhenlei520) [reply src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.Azure/README.md]: processed

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.FileSystem/README.md (hunk 2 above)
Comment (zhenlei520) [reply src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.FileSystem/README.md]: processed

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.MySql/README.md (hunk 3 above)
Comment (zhenlei520) [reply src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.MySql/README.md]: processed

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.Oracle/README.md (hunk 4 above)
Comment (zhenlei520) [reply src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.Oracle/README.md]: processed

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.PostgreSql/README.md (hunk 5 above)
Comment (zhenlei520) [reply src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.PostgreSql/README.md]: processed

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.Redis/README.md (hunk 6 above)
Comment (zhenlei520) [reply src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.Redis/README.md]: processed

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.SqlServer/README.md (hunk 7 above)
Comment (zhenlei520) [reply src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.SqlServer/README.md]: processed

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.WaitHandles/README.md (hunk 8 above)
Comment (zhenlei520) [reply src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.WaitHandles/README.md]: processed

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.ZooKeeper/README.md (hunk 9 above)
Comment (zhenlei520) [reply src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.ZooKeeper/README.md]: processed

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.Azure/README.zh-CN.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+```c#
+Install-Package Masa.Contrib.Data.DistributedLocking.Medallion.Azure
+```
+
+1. 修改类`Program`
+
+``` C#
+builder.Services.AddDistributedLock(medallionBuilder =>
+{
+    medallionBuilder.UseAzure("replace you connectionString", "replace you blobContainerName");
```
Comment (doddgu) [inline src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.Azure/README.zh-CN.md:16]: you -> your

**Review by doddgu — CHANGES_REQUESTED** [review]

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.Azure/README.zh-CN.md (hunk 12 above)
Comment (zhenlei520) [reply src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.Azure/README.zh-CN.md]: processed

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Local/README.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (zhenlei520) [reply src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Local/README.md]: processed

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Local/README.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (zhenlei520) [reply src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Local/README.md]: processed

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.Azure/README.md (hunk 1 above)
Comment (zhenlei520) [reply src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.Azure/README.md]: processed

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.MySql/README.md (hunk 3 above)
Comment (zhenlei520) [reply src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.MySql/README.md]: processed

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.PostgreSql/README.md (hunk 5 above)
Comment (zhenlei520) [reply src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.PostgreSql/README.md]: processed

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.SqlServer/README.md (hunk 7 above)
Comment (zhenlei520) [reply src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.SqlServer/README.md]: processed

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.Redis/README.md (hunk 6 above)
Comment (zhenlei520) [reply src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.Redis/README.md]: processed

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.WaitHandles/README.md (hunk 8 above)
Comment (zhenlei520) [reply src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.WaitHandles/README.md]: processed

File: src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.ZooKeeper/README.md (hunk 9 above)
Comment (zhenlei520) [reply src/Data/DistributedLocking/Masa.Contrib.Data.DistributedLocking.Medallion.ZooKeeper/README.md]: processed

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**